src/js/overlay.js

```diff
@@ -201,8 +201,6 @@ Overlay.prototype.show = function() {
 		document.body.appendChild(shadow);
 	}
 
-	this.content.focus();
```

Comment:
Oh, I understand. It's not obvious, but tab through the content with/without this focus and you'll see the change. It's basically setting the tab order into the content of the overlay so that a keyboard user can use it and screen readers know where to start reading.